test/eval.test.ts

```diff
@@ -101,7 +102,7 @@ describe('Env Evaluation', () => {
         }),
       ).toThrow(
         new ManifestEvaluatingError([
-          'Manifest env variable "deploy.env.foo" can not be mapped for "${{foo.}}" expression: Unexpected EOF [0,7]',
+          `Manifest env variable "deploy.env.foo" can not be mapped to "\${{foo.}}": ${new UnexpectedTokenError('.', 6).message}`,
```

Comment:
Now it is unclear what exactly is inside the error.
Let's use text in tests like that as much as it is possible.